src/datetime/addMinutes.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * To add minutes to a datetime.
+ * @param {Date} dateFrom
+ * @param {number} minutes
+ * @return Date
+ */
+
+export const addMinutes = (dateFrom: Date, minutes: number): Date => {
```

Comment:
Same considerations as with `addHours`. Consider using constants instead of literals, and consider calling `addMilliseconds` to avoid duplicated code.

Reply:
export const addMinutes = (dateFrom: Date, minutes: number): Date => {
  return new Date( dateFrom.getTime() + minutesToMilliseconds(minutes));
};